src/main/java/edu/kis/powp/jobs2d/drivers/transformation/Scale.java

```diff
@@ -0,0 +1,18 @@
+package edu.kis.powp.jobs2d.drivers.transformation;
+
+public class Scale implements PointTransformation {
+    private double deltaX;
+    private double deltaY;
```

Comment:
Is Flip necessary? Hor. and ver. flips can be instances of Scale (maybe factory would be useful)